### DISCUSSION TITLE
With JS SDK, how to set order to DESC?

### DISCUSSION THREAD

**Discussion by ilearnio** [pocketbase/pocketbase#2713] [Q&A]
I am trying to send a request using pocketbase JS SDK, but it seems like there is no way to pass it an order (for descending order) property. Here's my code:

```ts
const records = await collection.getList(1, 9999, {
    sort: 'created',
    order: 'desc'
})
```

So the "order" property is ignored. Also I cannot see any alternative for "order" in the `RecordListQueryParams` TypeScript type. Could you please advice? 

Thanks

PS: Using latest PB and JS SDK

**Comment by ganigeorgiev** [comment]
To set the sort order you can use `+` (default when not set) or `-` prefix, for example:

```js
// note that the max records are capped to 500, so even if you send 9999 we'll return max 500 in a single request
// alternatively to get all records at once (internally they will be batched), you can use getFullList({ sort: '...' })
const records = await collection.getList(1, 20, {
    // DESC created and ASC title
    sort: '-created,title',
})
```

**Reply by ilearnio** [reply]
Thank you so much!
